docs/content/developer/iota-move-ctf/challenge_1.mdx

```diff
@@ -0,0 +1,41 @@
+
+import ChallengeVerifier  from '@site/src/components/CTF/ctf-verifier';
+
+
+
+
+# Challenge 1: Checkin
+
+In this first challenge, your task is to interact with a basic Move contract. The contract defines a `Flag` object, which you need to retrieve by calling a specific function.
+
+The contract mints a new flag and transfers it to your account. Your goal is to [call the function](../getting-started/publish.mdx#accessing-your-package), capture the flag, and then submit the object ID to verify your success.
+
+This challenge is designed to be an easy introduction, guiding you through the process of interacting with the blockchain, calling functions, and understanding the basics of flag capture.
+
+
+## Deployed Contract Address:
+```
+Package: 0xed1bbeb067ea89b70859641557700997a39bf749b5eedf91e221d56a5d961622
+```
+
+## Contracts
```

Comment:
A line should be left between the two headings.

Reply:
```suggestion

## Contracts
```